plugin/trino-teradata/src/test/java/io/trino/plugin/teradata/integration/clearscape/TeradataHttpClient.java

```diff
@@ -0,0 +1,166 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.plugin.teradata.integration.clearscape;
+
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.DeserializationFeature;
+import com.fasterxml.jackson.databind.MapperFeature;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.json.JsonMapper;
+
+import java.io.IOException;
+import java.io.UncheckedIOException;
+import java.net.URI;
+import java.net.http.HttpClient;
+import java.net.http.HttpRequest;
+import java.net.http.HttpResponse;
+import java.util.concurrent.CompletableFuture;
+
+import static com.google.common.net.HttpHeaders.AUTHORIZATION;
+import static com.google.common.net.HttpHeaders.CONTENT_TYPE;
+
+public class TeradataHttpClient
+{
+    private static final String APPLICATION_JSON = "application/json";
+    private static final String BEARER = "Bearer ";
+
+    private final String baseUrl;
+    private final HttpClient httpClient;
+    private final ObjectMapper objectMapper;
+
+    public TeradataHttpClient(String baseUrl)
+    {
+        this(HttpClient.newBuilder().version(HttpClient.Version.HTTP_1_1).build(), baseUrl);
+    }
+
+    public TeradataHttpClient(
+            HttpClient httpClient,
+            String baseUrl)
+    {
+        this.httpClient = httpClient;
+        this.baseUrl = baseUrl;
+        this.objectMapper = JsonMapper.builder()
+                .configure(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES, false)
+                .configure(MapperFeature.ALLOW_FINAL_FIELDS_AS_MUTATORS, false)
+                .build();
+    }
+
+    // Creating an environment is a blocking operation by default, and it takes ~1.5min to finish
+    public CompletableFuture<EnvironmentResponse> createEnvironment(CreateEnvironmentRequest createEnvironmentRequest,
+            String token)
+    {
+        var requestBody = handleCheckedException(() -> objectMapper.writeValueAsString(createEnvironmentRequest));
```

Comment:
Avoid `var` https://github.com/trinodb/trino/blob/master/.github/DEVELOPMENT.md#avoid-var
Same for others.

Reply:
Thank you for the reference. Removed var usage across the Teradata connector plugin